### PR TITLE
image optimizer and fixing Save Tickets functionality

### PR DESCRIPTION
**Pull request by chrisjaimes**
* Fix Save Tickets 🙌 
* Added Sharp library for Image optimization in Production mode
* Removing unused files

### CODE REVIEW

File: app/payment/success/[confirmation]/[ticketId]/page.tsx [hunk 1]
```diff
@@ -78,14 +78,7 @@ const IndividualTicketPage = ({
...
       doc.setFontSize(22);
-      doc.addImage(
-        '/_next/static/media/logo.53eaf456.png',
-        'PNG',
-        70,
-        20,
-        70,
-        50
-      );
+      doc.addImage(PtLogo.src, 'PNG', 70, 20, 70, 50);
```
Comment (chrisjaimes) [inline app/payment/success/[confirmation]/[ticketId]/page.tsx:81]: Fix save tickets

File: app/payment/success/[confirmation]/page.tsx [hunk 2]
```diff
@@ -128,14 +128,7 @@ const PaymentSuccessPage = ({
...
       doc.setFontSize(22);
-      doc.addImage(
-        '/_next/static/media/logo.53eaf456.png',
-        'PNG',
-        70,
-        20,
-        70,
-        50
-      );
+      doc.addImage(PtLogo.src, 'PNG', 70, 20, 70, 50);
```
Comment (chrisjaimes) [inline app/payment/success/[confirmation]/page.tsx:131]: Fix save tickets

File: package.json [hunk 3]
```diff
@@ -46,6 +46,7 @@
     "react-redux": "^9.1.1",
     "react-router-dom": "^6.22.0",
     "redux-persist": "^6.0.0",
+    "sharp": "^0.33.3",
```
Comment (chrisjaimes) [inline package.json:49]: Image optimizer lib for production mode

File: app/payment/success/[confirmation]/page.tsx [hunk 4]
```diff
@@ -81,13 +81,20 @@ const PaymentSuccessPage = ({
...
+      // TODO: do not use alert, show message
+      const message = await response.text();
+      if (response.status == 400) {
+        // bad request
+        alert(message);
+      } else if (response.status == 404) {
+        // not found
+        alert(message);
+      } else {
+        alert("We're sorry, something went wrong");
```
Comment (chrisjaimes) [inline app/payment/success/[confirmation]/page.tsx:93]: Catch backend response in Ticket Share functionality

**Review by dnsantosuosso — APPROVED** [review]

**Pull request merged**